Cargo.toml

```diff
@@ -23,7 +23,7 @@ serde = { version = "^1.0", features = ["derive"] }
 serde_json = { version = "^1.0" }
 
 # Optional dependencies
-socks = { version = "0.3", optional = true }
```

Comment:
I don't think I can publish to crates.io with a local dependency. I think you need to move `socks` inside our `src` folder and include it as a local module

Reply:
ahah ooops

Reply:
`socks` included in `src`